### DISCUSSION TITLE
Use Bundle project with Nerdbank.GitVersioning

### DISCUSSION THREAD

**Discussion by maniglia** [wixtoolset/issues#7322] [Questions]
Is there a way to use Nerdbank.GitVersioning in Bundle project?
If I try to compile a project of type Bundle I get the following error:
_C:\Users\maniglia\.nuget\packagesnerdbank.gitversioning\3.5.119\build\Nerdbank.GitVersioning.targets(122,5): error : CodeDomProvider not available for language: wix. No version info will be embedded into assembly._

**Comment by robmen** [comment]
Honestly, that sounds like a good question for Nerdbank.GitVersioning.

**Reply by maniglia** [reply]
You are right, I have analyzed the sources of
Nerdbank.GitVersioning and I think there is a bug in the [AssemblyVersionInfo.cs](https://github.com/dotnet/Nerdbank.GitVersioning/blob/6d30af47025b6af6e094e6054b67dc12a18b054f/src/Nerdbank.GitVersioning.Tasks/AssemblyVersionInfo.cs) file, because he doesn't know how to manage the _wix_ CodeLanguage.

**Comment by rseanhall** [comment]
Seems like they had a regression of https://github.com/dotnet/Nerdbank.GitVersioning/issues/53. You can workaround it with https://github.com/dotnet/Nerdbank.GitVersioning/issues/292.

**Reply by maniglia** [reply]
I did some tests, with Nerdbank.GitVersioning 3.5.119 and 3.3.37.
Both types of projects: _Bundle (.exe)_ and _Package (.msi)_ have the property _Language_ set to _wix_, but only in case of _Bundle_ i get the error _CodeDomProvider not available for language: wix._.
I also tried setting the msbuild property _NBGV_EmitThisAssemblyClass_ to _false_, without any effect.

**Comment by maniglia** [comment]
- This is the [PR](https://github.com/dotnet/Nerdbank.GitVersioning/pull/918) for Nerdbank.GitVersioning to improve compatibility with WiX Toolset V4.
- This is the [Issue](https://github.com/dotnet/Nerdbank.GitVersioning/issues/913#issuecomment-1499983945) which contains a workaround.

If my PR is accepted, the end user will be able to use Nerdbank.GitVersioning with WiX projects without having to do anything else (I hope).

**Reply by BMurri** [reply]
I noticed that your PR was accepted and merged today, so I would expect your fix to be in their next release.
